examples/nextjs/pages/markets/[marketAccount]/place-bet.tsx

```diff
@@ -40,12 +49,12 @@ const PlaceBet = () => {
         try {
             const marketResponse = await getMarket(program, marketAccount);
             marketResponse.data.account.mintAccount
-            setMarket(marketResponse.data.account);
-            const marketOutcomeAccountsResponse = await getMarketOutcomeAccounts(program, marketAccount, marketResponse.data.account.marketOutcomes);
+            setMarket(marketResponse.data);
+            const marketOutcomeAccountsResponse = await getMarketOutcomesByMarket(program, marketAccount);
             setMarketOucomes(marketOutcomeAccountsResponse.data.marketOutcomeAccounts);
-            const defaultFormState = marketResponse.data.account.marketOutcomes.reduce((formState, marketOutcome) => ({
+            const defaultFormState = marketOutcomeAccountsResponse.data.marketOutcomeAccounts.reduce((formState, { account: { title }}) => ({
                 ...formState,
-                [marketOutcome]: {
+                [title]: {
                     backing: "back",
```

Comment:
I think you'll need to update this default form state to use the new `forOrAgainst` key?